cypress.config.js

```diff
@@ -0,0 +1,20 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    supportFile: false,
+    specPattern: 'cypress/integration/**/*.spec.{js,ts,jsx,tsx}',
```

Comment:
The 'specPattern' seems to have a syntax error with the wildcard character. It should be '**/*' to correctly match any subdirectories. The correct pattern might be 'cypress/integration/**/*.spec.{js,ts,jsx,tsx}'.